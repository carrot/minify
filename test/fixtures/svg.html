<svg width="178px" height="297px" viewBox="0 0 178 297" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <title>Slice 1</title>
    <desc>Created with Sketch.</desc>
    <defs></defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <circle id="Oval" stroke="#979797" fill="#D8D8D8" cx="45" cy="252" r="44"></circle>
        <circle id="Oval-Copy" stroke="#979797" fill="#D8D8D8" cx="133" cy="252" r="44"></circle>
        <ellipse id="Oval-2" stroke="#979797" fill="#D8D8D8" cx="89.5" cy="113" rx="29.5" ry="112"></ellipse>
    </g>
</svg>
