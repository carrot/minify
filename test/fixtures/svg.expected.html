<svg width="178" height="297" viewBox="0 0 178 297" xmlns="http://www.w3.org/2000/svg"><title>Slice 1</title><g fill="#D8D8D8" fill-rule="evenodd" stroke="#979797"><circle cx="45" cy="252" r="44"></circle><circle cx="133" cy="252" r="44"></circle><ellipse cx="89.5" cy="113" rx="29.5" ry="112"></ellipse></g></svg>
